truss/cli/cli.py

```diff
@@ -88,7 +88,7 @@ def image():
     "--skip-confirm",
     is_flag=True,
     show_default=True,
-    default=False,
+    default=True,
```

Comment:
can we just remove this option altogether? there's an `if` statement to remove below & that's all you need to do

Reply:
I'm down to remove it completely, will do that rn.